### PR TITLE
RST-2390 Moving fuse_rl to fuse_models

### PR DESCRIPTION
**Pull request by ayrton04**
FYI @efernandez

### CODE REVIEW

File: fuse_models/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package fuse_models
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.2.0 (2019-07-12)
+------------------
+* Get predict_to_current_time ROS param (`#17 <https://github.com/locusrobotics/fuse_rl/issues/17>`_)
```
Comment (ayrton04) [inline fuse_models/CHANGELOG.rst:13]: I left the changelog links pointing to the `fuse_rl` repo so that those PRs can still be viewed.

**Review by svwilliams — APPROVED** [review]
I reviewed every single line. It looks perfect.

**Comment by ayrton04** [comment]
You actually _did_ review every single line, at least at some point. I'll start moving these classes...

File: fuse_models/package.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>fuse_models</name>
+  <version>0.2.0</version>
```
Comment (efernandez) [inline fuse_models/package.xml:4]: I think this version must match the one in the other packages in the repo.

**Comment by efernandez** [comment]
BTW @ayrton04 If you want to keep `fuse_rl` history, you could try something like this: https://stackoverflow.com/questions/41811986/git-move-directory-to-another-repository-while-keeping-the-history

Let me know if you decide to do so and need help.

**Comment by ayrton04** [comment]
Done.

**Comment by ayrton04** [comment]
BTW, before I pushed this, I did a `git diff origin/RST-2390-move-fuse-rl` and it came up empty (if you ignore the changelog fix). Should be in the same state.

**Pull request merged**